src/Bicep.LangServer/Providers/BicepCompilationProvider.cs

```diff
@@ -41,9 +41,9 @@ public CompilationContext Update(IReadOnlyWorkspace workspace, CompilationContex
             return this.CreateContext(syntaxTreeGrouping, modelLookup);
```

Comment:
Do you need to transfer the previous `configHelper` here?

FWIW, I generally feel like default parameters are worth avoiding in C#, as they can be very easy to miss when refactoring, and in this case - it's very important to get it right. IMO it would be worth making `configHelper` a non-default param on the `CreateContext` method and `Compilation` constructor.

Reply:
Makes sense. Updated here:
https://github.com/Azure/bicep/commit/74db298612aac1a8c0d155fca8b08251f23cf1ae